avidemux_ru.ts

```diff
@@ -9565,7 +9565,7 @@ I will assume it has and try to continue</source>
     <message>
         <location filename="../../../avidemux_plugins/ADM_demuxers/Matroska/ADM_mkvIndexer.cpp" line="+630"/>
         <source>Indexing Matroska Video Track</source>
-        <translation type="unfinished"></translation>
+        <translation>Индексация видеодорожки Matroska</translation>
```

Comment:
Would it be a total no-go to use the original Russian word for Matroska? I can expand the compressed English source to its full meaning as "Indexing the video track in a Matroska container" (acutely missing either a preposition or congruence in the translation).

Reply:
Original Russian word isn't used at all across Russian speakers unfortunatly :/ Take a look at wiki article https://ru.wikipedia.org/wiki/Matroska — it is "Matroska" everywhere. Anyway I've rephrased the translation.